client/wfnews-war/src/main/webapp/config.jsp

```diff
@@ -121,9 +121,15 @@
       openmaps = openmaps.substring(0, openmaps.length() - 1); //Strip off trailing slash, if it exists.
     }
 
+    String services6 = EnvironmentVariable.getVariable("SERVICES6_BASE_URL"); 
```

Comment:
I think this needs to be moved above line 69, otherwise will it be defined there?